src/api/schema.js

```diff
@@ -402,6 +402,7 @@ const rootSchema = gql`
     deletePhoneNumbers(organizationId: ID!, areaCode: String!): JobRequest
     releaseCampaignNumbers(campaignId: ID!): Campaign!
     clearCachedOrgAndExtensionCaches(organizationId: String!): String
+    clearEntireRedisCache(adminPerms: Boolean): String
```

Comment:
adminPerms is not a required argument.  any falsy value will be unable to globally clear cache